magma/conversions.py

```diff
@@ -15,6 +15,7 @@
 from .tuple import Tuple, Product
 from .bitutils import int2seq
 import hwtypes
+import hwtypes as ht
```

Comment:
nit: stylistically, importing the same thing twice not preferred. think it's ok to refactor `hwtypes` to `ht` in the rest of this file if that's the preferred style going forward